demisto_sdk/commands/init/README.md

```diff
@@ -42,3 +46,8 @@ once given a script will be created under "Scripts" directory and the yml file w
 
 `demisto-sdk init --pack -n My_Pack`
 This will create a new pack named "My_Pack" under the "Packs" directory in content repo.
+
+`demisto-sdk init --contribution ~/Downloads/my_contribution.zip`
```

Comment:
is there a limitation that this need to be ran from the content repo root?

Reply:
It needs to be run from within the content repo but it doesn't matter from where in the repo.